app/assets/javascripts/controllers/angular-bootstrap/DatepickerCtrl.js

```diff
@@ -0,0 +1,22 @@
+miqAngularApplication.controller('DatepickerCtrl', function ($scope) {
+  $scope.minDate = new Date();
+
+  $scope.disabled = function(date, viewDate, mode) {
+    disabledDate = new Date($scope.$parent.angularForm[viewDate].$viewValue);
+
+    if($scope.$parent.angularForm[viewDate].$untouched) {
+      return ( mode === 'day' && (disabledDate.getUTCDate() === date.getDate()) &&
+                              disabledDate.getUTCMonth() === date.getMonth() &&
+                              disabledDate.getUTCFullYear() === date.getFullYear())
```

Comment:
Sure, I understand, just, are you sure it should do the UTC thing? Is it because you're sending just year, month and day without timezone and constructing disabledDate out of it? (I really hate time zones ;).)